### ISSUE TITLE
"make uninstall" support

### ISSUE THREAD

**Issue by diverger** [CNugteren/CLBlast#554]
Hi,

It seems I can't 'make uninstall' to uninstall the lib.

**Comment by CNugteren** [comment]
Indeed, this feature is not available. I don't think I have time myself to work on it, I'm open to accepting pull requests.
